app/scripts/modules/google/serverGroup/details/serverGroupDetails.gce.controller.js

```diff
@@ -246,6 +247,12 @@ module.exports = angular.module('spinnaker.serverGroup.details.gce.controller',
       }
     };
 
+    let processLabels = () => {
+      if (!Object.keys(this.serverGroup.instanceTemplateLabels).length) {
```

Comment:
Can `this.serverGroup.instanceTemplateLabels` be undefined / null here? I think `Object.keys` will throw an error if that's the case.